tests/posix/newlib/testcase.yaml

```diff
@@ -0,0 +1,4 @@
+tests:
+  portability.posix:
+    tags: posix
+    min_ram: 64
```

Comment:
You should exclude native_posix

Reply:
and the toolchain supporting newlib:

`filter: TOOLCHAIN_HAS_NEWLIB == 1`

Reply:
Thanks, updated.